PRIVACY.md

```diff
@@ -0,0 +1,21 @@
+# Semgrep Privacy Policy
+
+Semgrep collects usage data to help us improve the product. We send two types of data to r2c servers: scan data and findings data.
+
+## Scan data
+
+Scan data provide information on the environment and performance of Semgrep. They power dashboards, identify anomalies with the product, and are needed for billing. The classes of data included are:
+- Project identity (e.g. name, URL)
+- Scan environment (e.g. CI provider, OS)
+- Author identity (e.g. committer email)
+- Commit metadata (e.g. commit hash)
+- Scan metadata, including type of scan and scan parameters (e.g. files or branches to ignore)
+- Review and review-requester identifying data (e.g pull-request ID, branch, merge base, request author)
```

Comment:
Request author would also be PII. Can we include it as a hash for now?

Reply:
yes +1

Reply:
See comment above re. rendering webhooks.

Reply:
I saw them and added a positive comment